src/featureFlagService.ts

```diff
@@ -0,0 +1,59 @@
+import { FeatureFlagService, FeatureFlagProvider } from '@rudderstack/integrations-lib';
+import logger from './logger';
+
+export const productionFlags = [
+  {
+    key: 'enable-pinterest-advertising-tracking-mapping',
+    name: 'Pinterest Advertising',
```

Comment:
What is the purpose of this property? I don't find anything similar to this in the Flagsmith UI.